packages/framework-provider-aws/src/library/events-adapter.ts

```diff
@@ -25,16 +23,18 @@ export async function readEntityEventsSince(
   logger: Logger,
   entityTypeName: string,
   entityID: UUID,
-  since?: string
+  since?: string,
+  kind?: EventEnvelope['kind']
 ): Promise<Array<EventEnvelope>> {
   const fromTime = since ? since : originOfTime
+  const eventKind = kind ? kind : 'event'
```

Comment:
These two lines can be:
```suggestion
  const fromTime = since ?? originOfTime
  const eventKind = kind ?? 'event'
```